ml/validator.py

```diff
@@ -0,0 +1,49 @@
+import argparse
+import wandb
+import os
+
+from dotenv import load_dotenv
+
+from utils import load_model, evaluate_model, log_results_to_wandb
+from mockdata import get_mock_data
+
+
+# Parse command line arguments to choose the model
```

Comment:
use the existing parser
https://github.com/safe-scan-ai/cancer-ai-3/blob/main/cancer_ai/utils/config.py